components/sections/StatsSection.vue

```diff
@@ -4,13 +4,13 @@
       And we only launched
     </h2>
     <h1 class="text-center text-5xl font-bold text-gray-300">
-      3 weeks ago!
+      5 weeks ago!
     </h1>
     <div class="mx-auto max-w-screen-xl px-4 py-8 text-center lg:py-16 lg:px-6">
       <dl class="mx-auto grid max-w-screen-md gap-8 text-white sm:grid-cols-3">
         <div data-aos="fade-up-right" class="flex flex-col items-center justify-center">
           <dt class="mb-2 text-3xl font-extrabold md:text-4xl">
-            225+
+            234+
```

Comment:
can we do total, combined across all repos here? that wouild be >300 now 🚀

Reply:
Done in #21